zerver/lib/events.py

```diff
@@ -647,7 +647,8 @@ def name(sub: Dict[str, Any]) -> str:
             # Remove our user from the subscribers of the removed subscriptions.
             if include_subscribers:
                 for sub in removed_subs:
-                    sub['subscribers'] = [id for id in sub['subscribers'] if id != user_profile.id]
+                    sub['subscribers'] = sub['subscribers'].copy()
```

Comment:
(I put in this copy to be safe, but is it necessary?)

Reply:
It shouldn't be necessary.